doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_features.rst

```diff
@@ -0,0 +1,176 @@
+It is an ideal solution for 2.4 GHz wireless applications that have the highest requirements for performance, energy efficiency, security, and ease of use, enabling you to incorporate innovative features hard to productize today.
+nRF54H20 includes a multiprotocol radio, supporting Bluetooth Low Energy and IEEE 802.15.4-based protocols.
+
+High performance and low power
+==============================
+
+The nRF54H20 SoC has been designed to provide high compute performance in your application, while maintaining or improving the battery life compared to previous devices.
+It achieves this by integrating two Arm Cortex-M33 processors clocked up to 320 MHz, 2 MB of MRAM as non-volatile memory, 1 MB of RAM, a 10 dBm TX power multiprotocol radio, and state-of-the-art security features, in an asymmetric multi-core system, in which the different CPUs are optimized for various types of workloads.
+This configuration allows the SoC to achieve a lower power consumption and better performance compared to the usage of identical CPUs.
+nRF54H20 uses MRAM as non-volatile memory.
+Compared to flash memory, MRAM is more energy efficient and can be written to up to 86x faster.
+Compared to earlier devices, external interfaces have also been improved to support significantly higher data rates (up to 40x for USB and up to 8x for external memory).
+This does not come at the expense of power consumption, as both active and sleep power consumption have been improved as well.
+
+Highly integrated solution
+==========================
+
+The nRF54H20 SoC can help you simplify hardware designs by replacing multiple components, like an application MCU, an external flash, and a wireless SoC with one integrated SoC.
+nRF54H20’s excellent energy efficiency enables smaller batteries to be utilized, further reducing both the design size and cost.
+It also provides a single easy-to-use development environment without having to use multiple tools from different vendors.
+
+Best-in-class wireless support
+==============================
+
+The nRF54H20 SoC integrates a best-in-class 2.4 GHz radio that supports a wide range of wireless protocols.
+It supports Bluetooth Low Energy, LE Audio, Direction Finding, Bluetooth mesh, Thread, Zigbee and Matter.
+This includes all Bluetooth 5.3 features and the preparation to support upcoming Bluetooth specification versions.
+Additionally, nRF54H20 has improved sensitivity and robustness against interference.
+
+22 nm FD-SOI process node
+=========================
+
+Leveraging the latest advancements in semiconductor manufacturing, nRF54H20 is built using a 22 nm process from GlobalFoundries.
+Smaller process nodes typically result in higher-leakage currents.
+However, this is mitigated by leveraging Fully Depleted Silicon-on-Insulator (FD-SOI) process technology.
+FD-SOI reduces the size of the transistor channel compared to bulk CMOS, resulting in lower leakage currents.
+
+Security is key
```

Comment:
Maybe just me, but I think this title isn't so descriptive? Maybe something like:

```suggestion
Additional security features
```
or 
```suggestion
Improved security design
```